crates/key-server/src/master_keys.rs

```diff
@@ -134,7 +166,9 @@ impl MasterKeys {
         key_server_object_id: &ObjectID,
     ) -> anyhow::Result<&IbeMasterKey, InternalError> {
         match self {
-            MasterKeys::Open { master_key } => Ok(master_key),
+            MasterKeys::Open { master_key } | MasterKeys::Committee { master_key, .. } => {
```

Comment:
what if we already moved to the new committee?

Reply:
see todo in the description - this pr does not handle next committee to keep the diff small.